Naming/src/main/java/com/epam/engx/cleancode/naming/task5/FileExtensionChecker.java

```diff
@@ -3,17 +3,17 @@
 
 import com.epam.engx.cleancode.naming.task5.thirdpartyjar.Predicate;
 
-public class FileExtPred implements Predicate<String> {
+public class FileExtensionChecker implements Predicate<String> {
 
-    private String[] extns;
+    private String[] extensions;
 
-    FileExtPred(String[] extns) {
-        this.extns = extns;
+    FileExtensionChecker(String[] extensions) {
+        this.extensions = extensions;
     }
 
     @Override
     public boolean test(String fileName) {
```

Comment:
```suggestion
    public boolean isAllowed(String fileName) {
```

Reply:
Absolutely agree.
But the FileExtensionChecker class implements the Predicate interface, which included in the thirdpartyjar package - it closed for changes according to the conditions of the task. Class overrides method "test" of interface. So I didn't rename it.

Reply:
By the way, the Predicate interface name is also incorrect, since a built-in functional interface of the same name appeared with java8.